csharp/ql/src/semmle/code/csharp/frameworks/ServiceStack.qll

```diff
@@ -78,7 +78,21 @@ module Sources {
         }
     }
 }
-
+/** Flow Sinks for the ServiceStack framework */
+module Sinks {
+    private import semmle.code.csharp.security.dataflow.flowsinks.Remote
+  
+    /** RemoteFlow sinks for service stack */
+    class ServiceStackRemoteRequestParameter extends RemoteFlowSink {
+      ServiceStackRemoteRequestParameter() {
+        exists(MethodCall mc |
+          mc.getTarget().hasQualifiedName("ServiceStack.IRestClient.Get") and
```

Comment:
Why is this limited to `Get` requests? Wouldn't `Put`, `Post`, ... be sinks too? Also, the async variants could be handled similarly.

Reply:
Will Add.  Thx